### ISSUE TITLE
when speech_to_text library is active at that time flutter_tts library is inactive in IOS vice versa 

### ISSUE THREAD

**Issue by Sagar710-CS** [dlutton/flutter_tts#438]
## 🐛 Bug Report

when speech_to_text library is active at that time flutter_tts library is inactive in IOS vice versa.

**Platform:** 
- [ ] :iphone: iOS

**Comment by RajaramTech** [comment]
Yes i have faced the same issue. Tried hard with multiple settings but it is not working.
